lib/backend/test/suite.go

```diff
@@ -730,6 +736,19 @@ func (s *BackendSuite) Mirror(c *check.C, b backend.Backend) {
 	c.Assert(item.ID, check.Equals, originalID)
 }
 
+// MakeBinaryPrefix returns function that prepends unique,
+// not-guaranteed-to-be-UTF8 prefix to any key, used to make
+// test suite concurrent-run proof
+func MakeBinaryPrefix() func(k string) []byte {
```

Comment:
This seems unused.

Reply:
Detritus of an abandoned approach. Deleted.